Replication failed: value too long for type character varying(256)

**Actual behavior:**
Replication failed: value too long for type character varying(256)


**Steps to reproduce the problem:**

Configure a new replication rule:
  - Source resource filter name: `**`
  - Example image: `sysinfo_paas_base_stage/s2i-tomcat:8a72ab6611e787d24a7e9fd9ad51ab38caee69a9-centos-openjdk11-tomcat8`


**Versions:**
- harbor version: 1.8.1
- docker engine version: 18.09.7
- docker-compose version: 1.23.2, build 1110ad01

**Log files:**

```
==> core.log <==
Jul 10 16:22:55 172.18.0.1 core[57435]: 2019-07-10T14:22:55Z [ERROR] [/replication/operation/controller.go:102]: the execution 3 failed: failed to create task records for the execution 3: pq: value too long for type character varying(256)

```

--- a/src/replication/operation/flow/stage.go
+++ b/src/replication/operation/flow/stage.go
@@ -17,7 +17,6 @@ package flow
 import (
 	"errors"
 	"fmt"
-	"strings"
 	"time"
 
 	"github.com/goharbor/harbor/src/common/utils/log"
@@ -331,15 +330,16 @@ func getResourceName(res *model.Resource) string {
 	if meta == nil {
 		return ""
 	}
+	repositoryName := meta.Repository.Name
 	if len(meta.Vtags) == 0 {
-		return meta.Repository.Name
+		return repositoryName
 	}
 
-	if len(meta.Vtags) <= 5 {
-		return meta.Repository.Name + ":[" + strings.Join(meta.Vtags, ",") + "]"
+	if len(meta.Vtags) == 1 {
+		return repositoryName + ":[" + meta.Vtags[0] + "]"
 	}
 
-	return fmt.Sprintf("%s:[%s ... %d in total]", meta.GetResourceName(), strings.Join(meta.Vtags[:5], ","), len(meta.Vtags))
+	return fmt.Sprintf("%s:[%s ... %d in total]", repositoryName, meta.Vtags[0], len(meta.Vtags))
 }
 
 // repository:c namespace:n -> n/c

